src/Symfony/Component/DependencyInjection/ContainerAwareInterface.php

```diff
@@ -18,5 +18,10 @@
  */
 interface ContainerAwareInterface
 {
+    /**
+     * Sets the container.
+     *
+     * @param ContainerInterface|null $container A ContainerInterface instance or null
```

Comment:
We do not add such annotations when it does not add anything that cannot be read from the signature, so this should be removed.